### PR TITLE
Add gcloud and kubectl to airflow image

### PR DESCRIPTION
**Pull request by aabf**

### CODE REVIEW

File: airflow/current/Dockerfile [hunk 1]
```diff
@@ -11,3 +11,10 @@ RUN pip install \
  boto3==1.10.40 \
  pyathena==1.9.0 \
  pyyaml==5.3
+
+#Intall gcloud
```
Comment (mpasa) [inline airflow/current/Dockerfile:15]:
```suggestion
# Install gcloud
```

File: airflow/current/Dockerfile [hunk 2]
```diff
@@ -11,3 +11,10 @@ RUN pip install \
...
+ENV CLOUDSDK_INSTALL_DIR /usr/local/gcloud/
+RUN curl -sSL https://sdk.cloud.google.com | bash
+ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
+#Install kubectl
```
Comment (mpasa) [inline airflow/current/Dockerfile:19]: Just as a note, we usually leave 1 space after the comment `# comment`

**Review by mpasa — APPROVED** [review]
Just some formatting comments

**Review by mpasa — APPROVED** [review]

File: airflow/current/README.md [hunk 3]
```diff
@@ -19,3 +19,23 @@ Where `${GIT_COMMIT}` is the first seven characters of the git commit.
...
+## Environment Variables
+
+    If you want to run this image with gcloud authentication you should define the following environment variables 
+
+    |         ENV Var           |                    Description                   | Require | 
+    |---------------------------|--------------------------------------------------|---------|
+    | GCP_AIRFLOW_ACCOUNT_EMAIL | Service account email                            |    y    |
+    | GCP_AIRFLOW_SA_FILE_PATH  | Service account json file's path                 |    y    |
+    | GCP_K8S_LOCATION          | GCP kubernetes location (ex: "us-east4gcpcore0") |    y    |
+    | GCP_K8S_ZONE              | GCP kubernetes zone (ex: "us-east")              |    y    |
```
Comment (mpasa) [inline airflow/current/README.md]:
```suggestion
    | GCP_K8S_ZONE              | GCP kubernetes zone (ex: "us-east4")              |    y    |
```

File: airflow/current/entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+while getopts ":hwsig" opt; do
+  case ${opt} in
+    h )
+      echo "Usage:"
+      echo "    docker run <image> -h          Display this help message."
+      echo "    docker run <image> -w          Start airflow as a webserver"
+      echo "    docker run <image> -s          Start airflow as a scheduler"
+      echo "    docker run <image> -w -g       Start airflow as a webserver with gcloud creadentials"
+      echo "    docker run <image> -s -g       Start airflow as a scheduler with gcloud creadentials"
```
Comment (mpasa) [inline airflow/current/entrypoint.sh:24]:
```suggestion
      echo "    docker run <image> -w -g       Start airflow as a webserver with gcloud credentials"
      echo "    docker run <image> -s -g       Start airflow as a scheduler with gcloud credentials"
```

File: airflow/current/entrypoint.sh [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    s )
+      echo "Start as a scheduler"
+      OPERATOR="scheduler"
+      ;;
+    i )
+      echo "Start as a initdb"
+      OPERATOR="initdb"
+      ;;
+    g )
+      echo "Gcloud auth and creates kubeconfig"
```
Comment (mpasa) [inline airflow/current/entrypoint.sh:40]:
```suggestion
      echo "Authenticates Gcloud and creates a kube config file"
```

File: airflow/upgrade/entrypoint.sh [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+while getopts ":hwsig" opt; do
+  case ${opt} in
+    h )
+      echo "Usage:"
+      echo "    docker run <image> -h          Display this help message."
+      echo "    docker run <image> -w          Start airflow as a webserver"
+      echo "    docker run <image> -s          Start airflow as a scheduler"
+      echo "    docker run <image> -w -g       Start airflow as a webserver with gcloud creadentials"
+      echo "    docker run <image> -s -g       Start airflow as a scheduler with gcloud creadentials"
```
Comment (mpasa) [inline airflow/upgrade/entrypoint.sh:24]:
```suggestion
      echo "    docker run <image> -w -g       Start airflow as a webserver with gcloud credentials"
      echo "    docker run <image> -s -g       Start airflow as a scheduler with gcloud credentials"
```

File: airflow/upgrade/entrypoint.sh [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+    s )
+      echo "Start as a scheduler"
+      OPERATOR="scheduler"
+      ;;
+    i )
+      echo "Start as a initdb"
+      OPERATOR="initdb"
+      ;;
+    g )
+      echo "Gcloud auth and creates kubeconfig"
```
Comment (mpasa) [inline airflow/upgrade/entrypoint.sh:40]:
```suggestion
      echo "Authenticates Gcloud and creates a kube config file"
```

File: airflow/current/entrypoint.sh [hunk 8]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+function init_gcloud {
+  EMAIL=${GCP_AIRFLOW_ACCOUNT_EMAIL:?"Need to be set and non-empty"}
+  FILE=${GCP_AIRFLOW_SA_FILE_PATH:?"Need to be set and non-empty"}
+  LOCATION=${GCP_K8S_LOCATION:?"Need to be set and non-empty"}
+  ZONE=${GCP_K8S_ZONE:?"Need to be set and non-empty"}
+  PROJECT=${GCP_K8S_PROJECT:?"Need to be set and non-empty"}
+  
+  #Start gcloud auth
```
Comment (mpasa) [inline airflow/current/entrypoint.sh:10]: Can we standarize comments to `# comment`?

**Review by mpasa — CHANGES_REQUESTED** [review]

File: airflow/current/entrypoint.sh (hunk 8 above)
Comment (aabf) [reply airflow/current/entrypoint.sh:10]: yes. 👍

**Review by mpasa — APPROVED** [review]
Nice! 🚀

**Pull request merged**